nodejs/week1/src/backend/app.js

```diff
@@ -0,0 +1,76 @@
+const express = require("express");
+const app = express();
+
+// import data here
+const meals = require("./data/meals");
+const reviews = require("./data/reviews")
+const reservations = require("./data/reservations")
+
+// this is where you will be adding your routes
+app.get("/", async (request, response) => {
+  response.send("Meal Sharing Web App");
+});
+// meals
+/*Respond with the json for all the meals. For each meal, 
+if there are reviews matching it's meal ID, add these reviews 
+to each meal in the form of an array. For meals that do not have any reviews, 
+the "reviews" property will be an empty array.*/
+
+app.get('/meals', async(request, response) => {
+  const mealsReviews = meals.map((meal) => {
+    meal.reviews = reviews.filter((review) => review.mealId === meal.id);
+    return meal;
+  });
+  response.send(mealsReviews);
+});
+
+//Respond with the json for all the meals (including it's reviews)
+// that are cheap (you define what a cheap meal is)
+app.get("/cheap-meals", async (request, response) => {
+  const cheapMeals = meals
+  .filter(meal => meal.price <= 50)
+  .map((meal)=> {meal.reviews = reviews
+  .filter((review)=> review.mealId === meal.id)
```

Comment:
Minor but try to format your code better, it'd be easier to read where each filter and map is used if it was like this:
```
const cheapMeals = meals
    .filter(meal => meal.price <= 50)
    .map((meal) => {
      meal.reviews = reviews
        .filter((review) => review.mealId === meal.id)
      return meal;
    })
```

or ```meal.reviews = reviews.filter((review) => review.mealId === meal.id)```
You can simply use the built in one in [VSCode](https://code.visualstudio.com/docs/editor/codebasics#_formatting).